extension/options/index.html

```diff
@@ -105,6 +105,23 @@ <h3 data-i18n="display_options_title"></h3>
 		<span data-i18n="hide_closed_issues"></span>
 	</label>
 
+	<h3 data-i18n="preload_pages_count"></h3>
+	<label>
+		<select name="preloadPagesCount">
+			<option value="1">1</option>
+			<option value="2">2</option>
+			<option value="3">3</option>
+			<option value="4">4</option>
+			<option value="5">5</option>
+			<option value="6">6</option>
+			<option value="7">7</option>
+			<option value="8">8</option>
+			<option value="9">9</option>
+			<option value="10">10</option>
+			<option value="11">11</option>
+		</select>
+	</label>
+
```

Comment:
Ok. I fixed the options UI in 25368e7.
Fetch my commit on your branch and then run `npm install` again.